test_ci_backend.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+# run django migrations check to ensure that there are no migrations left to create
+cd src || exit
```

Comment:
exit 1, to guarantee that it exists with an error code

Reply:
New commit with the requested change applied.